context.go

```diff
@@ -351,18 +351,14 @@ func (ctx *Context) InternalError(err error, message string) error {
 // RawError reports an error wrapped in a message.
 func (ctx *Context) RawError(err error, message string) error {
 	wrapped := pkgerrors.Wrap(err, message)
-	return ctx.error(ctx.getEvaluatedFields(), wrapped, wrapped, wrapped)
+	return ctx.error(ctx.getEvaluatedFields(), err, errors.New(message), wrapped)
```

Comment:
I am not sure why this needs 3 errors. The original error (the param name could be better on the function), internal, safe. But that is independent of this change, so... 🤷

Reply:
Yeah - I don't think that `internal` and `safe` should really need to be errors, although I think it's been done like this so that `safe` could just be returned from the `error` method.